### ISSUE TITLE
Valve doesn't work until broken and re-placed

### ISSUE THREAD

**Issue by thefiz** [McJty/DeepResonance#170]
The valve from deep resonance seems to like to lock up until I break it and re-place it.  Once I do this, without changing any settings on the valve, it works fine.

I'm running 1.4.2

**Comment by McJty** [comment]
This has been reported before but is most likely not a bug in Deep Resonance but some weird chunkloading interaction caused by another mod. Other mods also suffer from this from time to time. As of now no idea what's going on

**Comment by thefiz** [comment]
What's interesting is another valve 1 block over and up works fine 100%. So I'm baffled.

**Comment by McJty** [comment]
Same chunk?

**Comment by jeridan** [comment]
My valve won't work until I lower the allowed % of all 3 values.  I have used a stack of nether stars after reaching 100% in Strength and Efficiency and a stack of emeralds after reaching 100% in Purity and it still wouldn't move the RCL until I lowered the % of all 3.

**Comment by McJty** [comment]
Can you clarify that a bit? What values does it not work at and what values does it work at?

**Comment by jeridan** [comment]
I have all 3 values set at 100% in the valve.  After getting 100% in Strength and Efficiency I used another stack of nether stars.  Then I got Purity to 100% and used another stack of emeralds and it would not go through the valve until I dropped all 3 values to 99%.  Each crystal that gets created says 100% on all three values so that's what I'm lost at.

**Comment by McJty** [comment]
Yes that's normal. The numbers are not accurate due to how floating point calculations work. Not much I can do about that

**Comment by thefiz** [comment]
I missed your reply. Same chunk yes

**Comment by jeridan** [comment]
Then perhaps a scanner/tool of some sort to accurately see the % of the RCL so we don't waste materials?  I have made several crystals that WAILA and rftools monitors said were 100% to all only to have the crystal come out 99.99 % on one of the stats.  Since the power output and overall power produced seems to take such a big hit with a .01% difference I'd prefer not to waste the RCL or the materials improving it.

**Comment by Locriste** [comment]
my valve also won't work if i don't set it to a max of 99% even though i want them at 100%. it worked yesterday but now it stopped i tried everything i know (i don't really know the mod) but it doesn't work

**Comment by Locriste** [comment]
nervermind i updated my modpack and it's working great now
